pandas/tests/io/formats/test_latex.py

```diff
@@ -0,0 +1,95 @@
+import pytest
```

Comment:
I think this should go in `pandas/tests/io/formats/test_to_latex.py`

Reply:
Moved those tests into ``test_to_latex.py``